src/components/Summary/StarRatingDisplay.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+import HelpIcon from '@mui/icons-material/Help';
+import ToolTip from '@mui/material/Tooltip';
+import { Grid, Typography, Rating } from '@mui/material';
+
+function StarRatingDisplay({ activeMeasure }) {
+  const starsTip = 'Star rating subject to change depending on measures and other resources. For more information, please contact NCQA.';
+
+  function ratingPresentUI() {
+    return (
+      <>
+        <Grid className="rating-trends__header-align">
+          <Typography variant="h3" className="rating-trends__h3-header">
+            Star Rating
+          </Typography>
+          <ToolTip title={starsTip}>
+            <HelpIcon className="rating-trends__help-icon" fontSize="small" />
+          </ToolTip>
+        </Grid>
+        <Rating
+          className="rating-trends__star-rating"
+          name="read-only"
+          value={activeMeasure.starRating || 0}
+          precision={0.5}
+          readOnly
+        />
+        <Typography className="rating-trends__star-rating-label">
+          {activeMeasure.label && `(${activeMeasure.label})`}
+        </Typography>
+
+      </>
+
+    )
+  }
+  function ratingNotPresentUI() {
+    return (
+      <>
+        <Grid className="rating-trends__header-align">
+          <Typography variant="h3" className="rating-trends__h3-header">
+            Star Rating
+          </Typography>
+          <ToolTip title={starsTip}>
+            <HelpIcon className="rating-trends__help-icon" fontSize="small" />
+          </ToolTip>
+        </Grid>
+        <Typography className="rating-trends__not-applicable">
+          N/A
+        </Typography>
+        <Typography>
+          (Not Applicable)
+        </Typography>
+      </>
+    )
+  }
+
+  if (activeMeasure.starRating > 0) {
+    return ratingPresentUI()
+  }
+  return ratingNotPresentUI()
+}
+
+StarRatingDisplay.propTypes = {
+  activeMeasure: PropTypes.shape({
+    measure: PropTypes.string,
+    starRating: PropTypes.number,
+    label: PropTypes.string,
+  }),
+}
+
+StarRatingDisplay.defaultProps = {
+  activeMeasure: {
+    starRating: 0,
+    label: '',
+  },
+}
+export default StarRatingDisplay
```

Comment:
Tim, I'm actually not thrilled with this approach at all. I'd rather just put it back the way it was and decide between showing the StarRating or N/A on a ternary.